library/alloc/src/lib.rs

```diff
@@ -147,6 +147,7 @@
 #![feature(slice_ptr_get)]
 #![feature(slice_range)]
 #![feature(std_internals)]
+#![feature(str_as_str)]
```

Comment:
Assuming this is only for the new tests, it should be added to the earlier `cfg_attr(test, feature(...))` attribute instead.

Reply:
Done